rules/data/fsPackagesNames.json

```diff
@@ -0,0 +1,3 @@
+[
+    "fs", "node:fs", "fs/promises", "node:fs/promises", "fs-extra"
+]
```

Comment:
Can we move this into the rule file? There doesn’t appear to be a need for this to be external.